center_printer.go

```diff
@@ -38,7 +38,11 @@ func (p CenterPrinter) Sprint(a ...interface{}) string {
 
 	if p.CenterEachLineSeparately {
 		for _, line := range lines {
-			ret += strings.Repeat(" ", (GetTerminalWidth()-runewidth.StringWidth(RemoveColorFromString(line)))/2) + line + "\n"
+			if (GetTerminalWidth()-runewidth.StringWidth(RemoveColorFromString(line)))/2 < 1 {
```

Comment:
as `GetTerminalWidth()-runewidth.StringWidth(RemoveColorFromString(line)))/2` is reused on line 43 you could store the value in a variable to not have to re-calculate the value